spec/jobs/scheduled/create_holiday_events_spec.rb

```diff
@@ -84,6 +84,37 @@
     expect(CalendarEvent.exists?(username: frenchy.username)).to eq(false)
   end
 
+  context "when there are disabled holidays" do
```

Comment:
I added these tests (a lot of it copied from existing tests here), but for some reason I can't get them to pass locally. I keep getting "Discourse::InvalidAccess: can_create? failed" when I run the specs via `bundle exec rake autospec`, despite this, it looks like they're passing the build though. :raised_eyebrow: :confused:

Reply:
You're probably just missing `LOAD_PLUGINS=1` when running autospec. I pulled the branch locally and `LOAD_PLUGINS=1 bin/rspec plugins/discourse-calendar/spec` works fine. 👍

Reply:
Yup, that was it! I ran that command and it worked locally! Thanks!